lib/loader/index.js

```diff
@@ -88,7 +88,7 @@ function postInstantiate(extendedExports, instance) {
   const __pin = exports.__pin || F_NOEXPORTRUNTIME;
   const __unpin = exports.__unpin || F_NOEXPORTRUNTIME;
   const __collect = exports.__collect || F_NOEXPORTRUNTIME;
-  const __rtti_base = exports.__rtti_base || ~0; // oob if not present
+  const __rtti_base = exports.__rtti_base || F_NOEXPORTRUNTIME;
```

Comment:
`__rtti_base` is a pointer into memory (unlike those above, which are functions), while `F_NOEXPORTRUNTIME` is a function. What would be the error here?

Reply:
yeah, just figure out this. Need other approach